ci/tests/general/general_vdisk.py

```diff
@@ -201,7 +201,7 @@ def write_to_volume(vdisk=None, vpool=None, location=None, count=1024, bs='1M',
                 raise ValueError('File {0} does not exist on Storage Router {1}'.format(location, root_client.ip))
         if not isinstance(count, int) or count < 1:
             raise ValueError('Count must be an integer > 0')
-        root_client.run('dd conv=notrunc if=/dev/{0} of={1} bs={2} count={3}'.format(input_type, location, bs, count))
+        root_client.run(['dd', 'conv=notrunc', 'if=/dev/' + input_type, 'of=' + location, 'bs=' + bs, 'count=' + str(count)])
```

Comment:
root_client.run(['dd', 'conv=notrunc', 'if=/dev/{0}'.format(input_type), 'of={0}'.format(location), 'bs={0}'.format(bs), 'count={0}'.format(count)])